src/components/redux/reducers/index.jsx

```diff
@@ -1,16 +1,53 @@
 import { combineReducers } from "redux";
+import {
+  ADD_TO_CART,
+  DELETE_FROM_CART,
+  MODIFY_QUANTITY,
+  SUM_TOTAL_PRICE,
+  ADD_ONE_MORE_ITEM,
+  REMOVE_ONE_MORE_ITEM,
+  QUANTITY_IN_SINGLE_PRODUCT,
+} from "../actions/actionsTypes";
 
-const currentBoardIDReducer = (state = [], action) => {
+const addOrDeleteProductData = (state = [], action) => {
   switch (action.type) {
-    case "CURRENT_BOARD_ID":
+    case ADD_TO_CART:
+      return [...state, action.payload];
+    case DELETE_FROM_CART:
+      return action.payload;
+    case MODIFY_QUANTITY:
```

Comment:
shouldn't this modify the existing state to increment quantity in an object or something?

Reply:
Yes, its modifying the quantity of the item , and now we split it into a different reducer